doc/api/pymongo/event_listeners.rst

```diff
@@ -0,0 +1,6 @@
+:mod:`event_listeners` -- Example listeners
+===========================================
+
+.. automodule:: pymongo.event_listeners
+   :synopsis: Example event listeners.
```

Comment:
Suggested rewording - "A collection of simple listeners for monitoring driver events."